### PR TITLE
Post/compatibility table

### PR DESCRIPTION
**Pull request by ercohen14**
This is Ernesto's blog article about Rails versions that he asked me to finish.

### CODE REVIEW

File: _posts/2019-04-26-compatibility-table.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+and [Ruby](https://www.ruby-lang.org/en/) across different versions. In the
+process of upgrading really old applications to more modern versions of Ruby and
+Rails we have ran into a lot of these combinations.
+
+<!--more-->
+
+<table id="ruby-rails-compatibility">
+  <thead>
+    <tr>
+      <td>Rails<br/>Version</td>
```
Comment (lubc) [inline _posts/2019-04-26-compatibility-table.md]: @ercohen14 You can link this to https://rubygems.org/gems/rails/versions

File: _posts/2019-04-26-compatibility-table.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      <td>Required<br/>Ruby<br/>Version</td>
```
Comment (lubc) [inline _posts/2019-04-26-compatibility-table.md]: @ercohen14 You can link Ruby version to https://www.ruby-lang.org/en/downloads/releases/

File: _posts/2019-04-26-compatibility-table.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      <td>Recommended<br/>Ruby<br/>Version</td>
+      <td>Required<br/>Rubygems<br/>Version</td>
```
Comment (lubc) [inline _posts/2019-04-26-compatibility-table.md]: @ercohen14 You can link https://rubygems.org/gems/rubygems-update/versions

**Comment by etagwerker** [comment]
@ercohen14 Every time you merge a post, please make sure you deploy it here: https://fastruby.io/blog

**Pull request merged**